src/main/java/rip/bolt/ingame/xml/MapFormatXMLParser.java

```diff
@@ -50,4 +58,14 @@ public static TournamentFormat parse(String name) {
         return null;
     }
 
+    public static List<String> getFilesFromFolder() {
+        File formatsFolder = new File(Tournament.get().getDataFolder(), "formats");
+        List<String> formatsList = new ArrayList<>();
+        if (formatsFolder.exists()) {
+            for (File child : formatsFolder.listFiles((file) -> file.getName().toLowerCase().endsWith(".xml"))) {
+                formatsList.add(child.getName().split(Pattern.quote("."))[0]);
```

Comment:
```suggestion
                formatsList.add(child.getName().substring(0, child.getName().lastIndexOf('.')));
```
If a file is called `stages.group.xml`, your current code will print `stages` instead of `stages.group`.